packages/befunge93/0.2.0/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+source ../../python/2.7.18/build.sh
+
+git clone -q 'https://github.com/programble/befungee' befunge93
```

Comment:
Can we checkout the exact tag?

Reply:
wdym

Reply:
Right now you are cloning master, which for all we know could be `v1.0.1`, so you need to pin an exact tag with `git checkout tag/v0.2.0`

Reply:
that is, after you clone